### ISSUE TITLE
Standalone clang's output has no syntax coloring

### ISSUE THREAD

**Issue by iahung** [msys2/MINGW-packages#17519]
`mingw-w64-x86_64-clang` and `mingw-w64-ucrt-x86_64-clang.

**Comment by Biswa96** [comment]
Duplicate of https://github.com/msys2/MINGW-packages/issues/2988

**Comment by iahung** [comment]
> Duplicate of #2988

No. It's not a duplicate of #2988. The clang compiler of CLANG64 (`mingw-w64-clang-x86_64-clang`) does have it output properly colored. So does the gcc compiler of MINGW64 and UCRT64. Let's call them the native compilers of the respective shells. Then the standalone clang compiler, that is not the native compiler of these shells, and only them, don't have their output colored.

**Comment by Biswa96** [comment]
In my case, clang++ shows color in both ucrt64 and clang64 environment.

1. clang++ in ucrt64

![1](https://github.com/msys2/MINGW-packages/assets/31443074/e739a163-02da-478e-9d30-91628cc8b349)

2. clang++ in clang64

![2](https://github.com/msys2/MINGW-packages/assets/31443074/f0323100-b7c6-47e7-8438-e2b223ff7536)

**Comment by iahung** [comment]
I have no idea why it doesn't work for me.
